tac/meeting-minutes/2025/2025-02-20.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: 2025-02-20
+parent: 2025 Meeting Minutes
+grand_parent: Meeting Minutes
+---
+
+![Antitrust Policy Notice](../images/antitrust-policy-notice.png "Antitrust Policy Notice")
+![All are Welcome in the LF Decentralized Trust Community](../images/all-are-welcome.png "All are Welcome in the LF Decentralized Trust Community")
+
+Linux Foundation Decentralized Trust is committed to creating a safe and welcoming community for all. For more information please visit our Code of Conduct: [LF Decentralized Trust Code of Conduct](../../governing-documents/code-of-conduct).
```

Comment:
Could we please update this link? 
It just needs a correction on the file name to "code-of-conduct.md"

Thank you!